js/admin.js

```diff
@@ -9,21 +9,24 @@
 
 	$( function() {
 
-		$( '.bfa-save-settings-button' ).on( 'click', function() {
+		$( '#bfa-settings-form' ).on( 'submit', function( event ) {
```

Comment:
See my comment above. Curious to understand if there are any benefits, specifically security-focused, to changing the form button/submit behavior. Not opposed, just trying to minimize the number of changes we're bundling in this hotfix.